### PR TITLE
Support nomad plan to take jobfile from stdin

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

File: command/plan.go [hunk 1]
```diff
@@ -86,12 +94,32 @@ func (c *PlanCommand) Run(args []string) int {
...
+	path := args[0]
+
+	var f io.Reader
+	switch path {
+	case "-":
+		if c.testStdin != nil {
+			f = c.testStdin
+		} else {
+			f = os.Stdin
+		}
```
Comment (dadgar) [inline command/plan.go]: can you set `path = "stdin"` and then change the error msg on :113 to be `Error opening file from %v..`

**Comment by dadgar** [comment]
LGTM

**Comment by nak3** [comment]
Thank you. Updated. 

> can you set path = "stdin" and then change the error msg on :113 to be Error opening file from %v..

You mean set `path = "stdin"` and it will change the error msg on L113? I hope I didn't misunderstand it.

**Comment by dadgar** [comment]
Yep exactly! Thanks!

**Pull request merged**